package.json

```diff
@@ -14,7 +14,9 @@
     "start": "eleventy --serve",
     "build": "ELEVENTY_ENV=production eleventy",
     "lint": "eslint --fix --ext .js",
-    "prettier": "prettier --write src/"
+    "prettier": "prettier --write src/",
+    "start:lambda": "netlify-lambda serve lambda_funcs",
```

Comment:
For these two new scripts, we should have documentation in the README that explains them. The docs should be written with dev in mind. And answer the questions like; "Do I need these for local dev?", "What do these do?", etc. Think about coming into this project with zero knowledge of how it's set up, what questions you might have. Doesn't have to be super in depth, and can/should link out to any external netlify docs as needed.

Reply:
Noted!